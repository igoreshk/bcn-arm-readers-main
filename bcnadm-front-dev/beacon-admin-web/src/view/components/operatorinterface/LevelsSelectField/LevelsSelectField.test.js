import React from 'react';
import { shallow } from 'enzyme';

import { LevelsSelectField } from './index';

const props = {
  buildings: [
    {
      address: 'address',
      description: 'description',
      entityId: 1,
      latitude: 1,
      longitude: 1,
      mapImage: null,
      mimiType: null,
      name: 'name',
      noImage: false
    }
  ],
  match: {
    params: {
      building: '',
      layer: ''
    }
  },
  translate: jest.fn()
};

const wrapper = shallow(<LevelsSelectField {...props} />);

describe('LevelsSelectField', () => {
  it('render connected(SMART) component', () => expect(wrapper.length).toEqual(1));
  it('snapshot for LevelsSelectField', () => expect(wrapper).toMatchSnapshot());
});
