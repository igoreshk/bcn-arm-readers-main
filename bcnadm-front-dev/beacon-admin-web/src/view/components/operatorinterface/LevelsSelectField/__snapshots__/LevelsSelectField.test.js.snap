// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LevelsSelectField snapshot for LevelsSelectField 1`] = `""`;
